alphagov/pay-publicapi
# Improve comments used for generating OpenAPI/Swagger file

## What you did
This pull request updates the `values` and `descriptions` in comments that are used to generate our OpenAPI/Swagger file. The updates bring descriptions of the API in line with our public tech docs at https://docs.payments.service.gov.uk/#gov-uk-pay-documentation, so they're clearer and more consistent.

(While doing this work, I also identified areas of the OpenAPI file where we can make future improvements. For example where descriptions are missing, other OpenAPI fields such as `example` need updating, and links definitions need updating. But to keep this PR in scope, we should look at those in a future iteration.)

## How to test
Please check if factually accurate.

Please bear in mind that:

- Style and tone is mostly copied or based on existing live content in https://docs.payments.service.gov.uk/#gov-uk-pay-documentation, so if a description needs major improvement it's best to save that for a future iteration (so we can update the tech docs to match) 
- I've intentionally removed min/max and default information from `description` fields so we don't duplicate what's already in the Swgger `min`, `max` and `default` objects
